.github/workflows/smoke-test.yml

```diff
@@ -7,7 +7,7 @@ on:
       - master
 
 env:
-  TIMEOUT: 3m
+  TIMEOUT: 10m
```

Comment:
this seems quite long. Can we revert it back to 3m please? Usually, we don't need to edit this value, if the chart isn't available after this time which means there's an error or chart is unstable.